x-pack/test/detection_engine_api_integration/security_and_spaces/group1/install_prebuilt_rules.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from 'expect';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import {
+  deleteAllRules,
+  deleteAllTimelines,
+  deleteRule,
+  getPrebuiltRulesStatus,
+  installPrebuiltRulesFromMocks,
+} from '../../utils';
+import {
+  ELASTIC_SECURITY_RULE_ID,
+  RulesPackageType,
+} from '../../utils/create_prebuilt_rule_saved_objects';
+import { deleteAllPrebuiltRules } from '../../utils/delete_all_prebuilt_rules';
+import { incrementPrebuiltRuleVersion } from '../../utils/increment_prebuilt_rule_version';
+import { installPrebuiltRules } from '../../utils/install_prebuilt_rules';
+
+const PACKAGE_TYPES: RulesPackageType[] = ['historical', 'latest_only'];
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const es = getService('es');
+  const supertest = getService('supertest');
+  const log = getService('log');
+
+  describe('install_prebuilt_rules', () => {
+    beforeEach(async () => {
+      await deleteAllRules(supertest, log);
+      await deleteAllTimelines(es);
+      await deleteAllPrebuiltRules(es);
+    });
+
+    PACKAGE_TYPES.forEach((packageType) => {
+      describe(`package type:  ${packageType}`, () => {
+        it('should install prebuilt rules', async () => {
+          const response = await installPrebuiltRulesFromMocks(supertest, es, packageType);
+
+          expect(response).toEqual(
+            expect.objectContaining({
+              rules_installed: expect.any(Number),
+              rules_updated: expect.any(Number),
+              timelines_installed: expect.any(Number),
+              timelines_updated: expect.any(Number),
+            })
+          );
+          expect(response.rules_installed).toBeGreaterThan(0);
```

Comment:
I think we should know how many prebuilt rules we install when calling `installPrebuiltRulesFromMocks`, and check for this exact number here.